lib/node.js

```diff
@@ -118,6 +118,7 @@ declare class Buffer extends Uint8Array {
 
 declare module "buffer" {
   declare var kMaxLength: number;
+  declare export var Buffer: typeof Buffer;
```

Comment:
Any suggestions for alternatives? I tested this and it appears to work correctly.